chapter_optimization/minibatch-sgd_vn.md

```diff
@@ -17,13 +17,18 @@ Stochastic Gradient Descent is not particularly *computationally efficient* sinc
 This suggests that there might be a happy medium, and in fact, that's what we have been using so far in the examples we discussed.
 -->
 
-*dịch đoạn phía trên*
+Đến phần này, ta đã tiếp xúc với hai hướng tiếp cận hàng đầu trong các phương pháp học dựa theo gradient: :numref:`sec_gd` sử dụng toàn bộ tập dữ liệu để tính toán gradient và cập nhật từng tham số một.
```

Comment:
```suggestion
Đến phần này, ta đã tiếp xúc với hai hướng tiếp cận tới hạn trong các phương pháp học dựa theo gradient: :numref:`sec_gd` sử dụng toàn bộ tập dữ liệu để tính toán gradient và cập nhật từng tham số một.
```

Reply:
```suggestion
Đến phần này, ta đã tiếp xúc với hai thái cực trong các phương pháp học dựa theo gradient: :numref:`sec_gd` sử dụng toàn bộ tập dữ liệu để tính toán gradient và cập nhật từng tham số một.
```
@cuongvng Anh nghĩ sao về cách dịch này? `extreme` có thể dịch thành `thái cực`, ở đây nghĩa là 2 phương pháp hoàn toàn khác nhau.
Tiện thể em xin lỗi vì ghi nhầm tên anh :((

Reply:
ok ko sao =))). `thái cực` mình nghĩ ổn hơn đấy 👍.